src/pages/home-page.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+import CityCard from "../components/CityCard";
+import Pollution from "../components/Pollution";
+import {useDispatch, useSelector} from 'react-redux'
+import {selectAllCountries,changeContinent} from '../redux/countriesSlice'
+
+const HomePage = () => {
+
+  let dispatch = useDispatch()
+const countriesArray = useSelector(selectAllCountries)
+// console.log("All Countries", countriesArray)
+
+const [continent, setContinent] = useState("All")
+
+const handleContinentChange=(continent)=>{
+  // console.log("Button clicked")
+  // setContinent(continent)
+  dispatch(changeContinent(continent))
+}
+
+
+return (
+  <>
+     <button onClick={()=>handleContinentChange("All")}>All</button>
+     <button onClick={()=>handleContinentChange("Africa")}>Africa</button>
+     <button onClick={()=>handleContinentChange("Asia")}>Asia</button>
+     <button onClick={()=>handleContinentChange("Europe")}>Europe</button>
+     <button onClick={()=>handleContinentChange("Oceania")}>Ocenia</button>
+     <button onClick={()=>handleContinentChange("Americas")}>Americas</button>
+
+
+  <div className="header">
+  <h2>Air Pollution</h2>
+  </div>
+
+  <div className="container">
+    {
+
+   countriesArray.map((country)=>{
+    // console.log(country)
+  return  <CityCard   key={country.name} name={country.name} continent={country.continent} flag={country.flag} lat={country.lat} lon={country.lon}/>
+})
+    }
+  </div>
+  </>
+)
+}
+
+export default HomePage;
```

Comment:
- [ ] As stated in the project requirement You should follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including: Colors (select one main color for your website), Typography: font face, size and weight, Layout: composition and space between elements. But I notice that your design is a little bit different from the design guidelines Kindly i recommend you to follow the design guidelines for the Homepage and adjust  **Colors (select one main color for your website**) you used 2 color for the body(rgb(221, 211, 211)) and the container(rgb(240, 91, 116)) and **Layout: composition and space between elements.** Just like the design guidelines 
- For the Home page the design guidelines has the following Layout: composition and space between elements, the category are adjust in the 2 columns with different color composition 
![Capture](https://user-images.githubusercontent.com/93598189/226131114-41e06719-0114-4b78-b4a2-42ffe9468d2a.PNG)

- Your Project design Look likes this 

![Capture](https://user-images.githubusercontent.com/93598189/226131303-821cd905-f517-4e95-84ea-9dd501e457c9.PNG)